.github/workflows/linting.yml

```
@@ -0,0 +1,33 @@
+name: Linting
+
+on: pull_request
+
+env:
+  TERRAFORM_VERSION: "1.2.4"
+
+jobs:
+  linting:
+    name: Linting
+
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 5
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
+        uses: actions/checkout@v2
```

We can upgrade this to use v3:
```suggestion
        uses: actions/checkout@v3
```

---

Fixed in https://github.com/nimblehq/nimble-growth-39-k8s-aws-eks/pull/11/commits/b9bbc7d7b24240d5064d3112d6be84f575503143 @hoangmirs